themes/ddbasic/sass/components/node/news.scss

```diff
@@ -0,0 +1,307 @@
+@import '../../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?